app-emulation/winetricks/winetricks-20150416.ebuild

```diff
@@ -1,19 +1,18 @@
 # Copyright 1999-2015 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
-# $Header: /var/cvsroot/gentoo-x86/app-emulation/winetricks/winetricks-20141130.ebuild,v 1.2 2015/03/23 05:34:46 tetromino Exp $
+# $Header: $
 
 EAPI=5
 
 inherit gnome2-utils eutils
 
 if [[ ${PV} == "99999999" ]] ; then
-	EGIT_REPO_URI="https://github.com/Winetricks/winetricks.git"
+	EGIT_REPO_URI="git://github.com/Winetricks/${PN}.git"
 	inherit git-r3
 	SRC_URI=""
 else
-	SRC_URI="http://winetricks.org/download/releases/${P}.tar.gz"
+	SRC_URI="https://github.com/Winetricks/${PN}/archive/${PV}.zip"
```

Comment:
and please also use "-> ${P}.tar.gz" to not have a distfile with only a version number as the name